lib/feed/view/feed_page.dart

```diff
@@ -301,54 +297,10 @@ class _FeedViewState extends State<FeedView> {
-                                  transitionBuilder: (child, animation) {
-                                    return FadeTransition(
-                                      opacity: Tween<double>(begin: 0.0, end: 1.0).animate(
-                                        CurvedAnimation(parent: animation, curve: const Interval(0.5, 1.0)),
-                                      ),
-                                      child: SlideTransition(
-                                        position: Tween<Offset>(begin: const Offset(1.2, 0), end: const Offset(0, 0)).animate(animation),
-                                        child: SizeTransition(
-                                          sizeFactor: Tween<double>(begin: 0.0, end: 1.0).animate(
-                                            CurvedAnimation(
-                                              parent: animation,
-                                              curve: const Interval(0.0, 0.25),
-                                            ),
-                                          ),
-                                          child: child,
-                                        ),
-                                      ),
-                                    );
-                                  },
-                                  child: !queuedForRemoval.contains(postViewMedias[index].postView.post.id)
-                                      ? PostCard(
-                                          postViewMedia: postViewMedias[index],
-                                          communityMode: state.feedType == FeedType.community,
-                                          onVoteAction: (int voteType) {
-                                            context.read<FeedBloc>().add(FeedItemActionedEvent(postId: postViewMedias[index].postView.post.id, postAction: PostAction.vote, value: voteType));
-                                          },
-                                          onSaveAction: (bool saved) {
-                                            context.read<FeedBloc>().add(FeedItemActionedEvent(postId: postViewMedias[index].postView.post.id, postAction: PostAction.save, value: saved));
-                                          },
-                                          onReadAction: (bool read) {
-                                            context.read<FeedBloc>().add(FeedItemActionedEvent(postId: postViewMedias[index].postView.post.id, postAction: PostAction.read, value: read));
-                                          },
-                                          listingType: state.postListingType,
-                                          indicateRead: true,
-                                        )
-                                      : null,
-                                );
-                              },
-                              childCount: postViewMedias.length,
+                            FeedWidget(
```

Comment:
I would rename this to `FeedPostWidget` or `FeedPostList` since the feed will eventually be able to hold comments as well (for user pages)!

Reply:
Good point! I went with `FeedPostList` since we already have similar (`PostCardList`) and I don't generally like putting another type name in a class name (`Widget`). Should be all set now.